idx/bigtable/bigtable.go

```diff
@@ -533,3 +539,38 @@ func (b *BigtableIdx) prune() {
 		}
 	}
 }
+
+// AddDefs adds defs to the index.
+func (b *BigtableIdx) AddDefs(defs []schema.MetricDefinition) {
+	pre := time.Now()
+
+	b.MemoryIndex.AddDefs(defs)
+
+	if b.cfg.UpdateBigtableIdx {
+		// Blocking write to make sure all get enqueued
+		for _, def := range defs {
+			b.writeQueue <- writeReq{recvTime: time.Now(), def: &def}
+		}
+	}
+
+	statControlRestoreDuration.Value(time.Since(pre))
+}
+
+// DeleteDefs deletes the matching key.
+func (b *BigtableIdx) DeleteDefs(defs []schema.MetricDefinition, archive bool) {
```

Comment:
seems we ignore the archive flag.
seems also we don't have any existing code to do archiving on bigtable. oops.
when a bigtable archive operation comes in, we should probably just log an error. and make sure that the control-server refuses to generate such messages

Reply:
right. ok so the control server can't do much, but i still think MT should generate errors upon bigtable archival requests.
the fallback to automatic deletes seems problematic. i'ld rather tell users to consider issuing a delete instead, so i proposed that change